### ISSUE TITLE
Activity doesn't work well with keyboard only controls

### ISSUE THREAD

**Issue by davefoord** [odeialba/moodle-mod_sortvoting#82]
It has just been pointed out to me that this activity doesn't work particularly well with keyboard controls (e.g. a screen reader user). I tried to navigate around the activity using just keyboard, and found it really hard to locate the draggable items - and then once I had found them, and moved an item, it then sent me back to the top of the page rather than keeping me in the draggable items bit.

It would be great if this could be set up to work with keyboard controls.

Thanks
